### PR TITLE
 Remove WorkFlow type argument.

### LINKED ISSUE #2140: Not to use WorkflowType to differentiate component, layout and behavior style in catalog app.
**Describe the Issue**

```
probably a follow-up. can we move away from this WorkflowType concept as I think it's obscure and making the code hard to understand. if we need a different style, we should just call it out.
```
https://github.com/google/android-fhir/pull/2088#discussion_r1290114147

**Would you like to work on the issue?**

### PR DESCRIPTION
**Pull request by santosh-pingle**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #2140 

**Description**
Remove workflow type argument value to differentiate the component, layout and behavior questionnaire.

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

**Type**
Choose one: Code health

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://github.com/google/android-fhir/wiki/Contributing) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [x] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

**Review by MJ1998 — APPROVED** [review]
LGTM

File: catalog/src/main/java/com/google/android/fhir/catalog/DemoQuestionnaireFragment.kt [hunk 1]
```diff
@@ -181,21 +184,29 @@ class DemoQuestionnaireFragment : Fragment() {
...
+  private fun getThemeId(title: String) =
+    if (
+      componentListViewModel.isComponent(requireContext(), title) ||
+        behaviorListViewModel.isBehavior(requireContext(), title)
+    ) {
+      R.style.Theme_Androidfhir_Component
+    } else if (layoutListViewModel.isDefaultLayout(requireContext(), title)) {
+      R.style.Theme_Androidfhir_DefaultLayout
+    } else {
+      R.style.Theme_Androidfhir_PaginatedLayout
```
Comment (jingtang10) [inline catalog/src/main/java/com/google/android/fhir/catalog/DemoQuestionnaireFragment.kt:196]: how different is paginated layout and component layout? can we merge them?

**Review by jingtang10 — CHANGES_REQUESTED** [review]

File: catalog/src/main/java/com/google/android/fhir/catalog/DemoQuestionnaireFragment.kt (hunk 1 above)
Comment (santosh-pingle) [reply catalog/src/main/java/com/google/android/fhir/catalog/DemoQuestionnaireFragment.kt:196]: done.

[the remark this answers is not in the document]
Comment (santosh-pingle) [reply catalog/src/main/java/com/google/android/fhir/catalog/DemoQuestionnaireFragment.kt]: done.

**Review by jingtang10 — APPROVED** [review]
thanks @santosh-pingle for this cleanup!

**Pull request merged**